packages/kit/types/index.d.ts

```diff
@@ -337,17 +337,52 @@ export interface ParamMatcher {
 	(param: string): boolean;
 }
 
+/**
+ * The interface representing an incoming request, used in `hooks.server.js`, `+page.server.js` and `+server.js` files
+ */
 export interface RequestEvent<
 	Params extends Partial<Record<string, string>> = Partial<Record<string, string>>
 > {
+	/**
+	 * An object for retrieving and modifying cookies. See [Cookies](https://kit.svelte.dev/docs/types#sveltejs-kit-cookies) for more info.
+	 */
 	cookies: Cookies;
+	/**
+	 * Gets the client's IP address.
+	 */
 	getClientAddress: () => string;
+	/**
+	 * Contains data populated from the [handle](https://kit.svelte.dev/docs/hooks#server-hooks-handle) hook.
+	 */
 	locals: App.Locals;
+	/**
+	 * An object of key-value pairs where each key is the name of a route parameter and each value is the value of that route parameter.
+	 * Example: If you access the route `src/routes/blog/[slug]/+page.svelte` through the URL `blog/hello-world`, `params` would be `{ slug: 'hello-world' }`
+	 */
 	params: Params;
+	/**
+	 * Contains [platform-specific context](https://kit.svelte.dev/docs/adapters#supported-environments-platform-specific-context).
+	 */
 	platform: Readonly<App.Platform>;
+	/**
+	 * The raw request object.
+	 */
 	request: Request;
+	/**
+	 * The route id of the current page - e.g. for `src/routes/blog/[slug]`, it would be `blog/[slug]`.
+	 */
 	routeId: string | null;
+	/**
+	 * If you need to set headers for the response, you can do so using the `setHeaders` method. This is useful for example if you want the page to be cached.
+	 *
+	 * Setting the same header multiple times is an error — you can only set a given header once.
+	 *
+	 * You cannot add a `set-cookie` header with `setHeaders` — use the [`cookies`](/docs/types#sveltejs-kit-cookies) API instead.
```

Comment:
We use a relative docs URL here but absolute URLs everywhere else. Should it be one or the other? I'd prefer relative URLs, but I'm not sure if we need to include the hostname for some other reason.

Reply:
Oversight - it should be absolute, so you can click on it when hovering the docs in your IDE